front/src/applications/operationalStudies/components/Study/AddOrEditStudyModal.tsx

```diff
@@ -79,58 +95,86 @@ export default function AddOrEditStudyModal({ editionMode, study, getStudy }: Pr
     setCurrentStudy({ ...currentStudy, tags: newTags });
   };
 
-  const createStudy = async () => {
-    if (!currentStudy?.name) {
+  const createStudy = () => {
+    if (!currentStudy) {
```

Comment:
```suggestion
    if (!currentStudy?.name) {
```
Here we need to check that the new study has at least a name